<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controlpid</class>
 <widget class="QWidget" name="controlpid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>531</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>PID Control</string>
   </property>
   <widget class="QRadioButton" name="Staticmode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Static Mode</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDoubleSpinBox" name="pDIff">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pstat">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>15.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pint">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;I&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;P&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;D&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="Variablemode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Variable Mode</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt; 350K</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>150K - 200K</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>300K - 350K</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="p350">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>32.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="p150">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>11.350000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="p250">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>22.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;P&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>250K - 300K</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>200K - 250K</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="p400">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>37.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="p300">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>27.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="p200">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>17.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt; 150 K</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Temperature&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="I300">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>1.375000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="I400">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>1.875000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="I200">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.875000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="I150">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.567000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="I250">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>1.125000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;I&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="I350">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>1.625000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="D300">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.917000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="D400">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>1.250000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="D200">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.583000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="D150">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.378000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="D250">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.750000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;D&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="D350">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>69</width>
       <height>26</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>1.083000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
